.github/chainguard/verify-github.sts.yaml

```diff
@@ -7,7 +7,7 @@ claim_pattern:
   job_workflow_ref: chainguard-dev/infra/.github/workflows/.terraform.yaml@.*
 
 permissions:
-  administration: read # to read the org's metadata
+  administration: write # to add/remove repositories
```

Comment:
why does verify need to write?

Reply:
we tested and saw differenteces whne running in CI and in my local machine and in your local

I will try that and see if the diff will disappear if not we revert to read

Reply:
please file an issue to track, or a comment?